supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -73,6 +73,16 @@ For example, you might combine existing product name attributes to create compou
 :name-spring-reactive: {name-runtime-spring-boot} with {name-runtime-vertx} reactive components
 ----
 
+[[non-breaking-spaces]]
```

Comment:
To be alphabetical, this should move above "Product names and versions"

Reply:
Fixed